frontends/bmd/src/pages/admin_screen.tsx

```diff
@@ -198,17 +198,17 @@ export function AdminScreen({
           {election ? (
             <p>
               <Text as="span" voteIcon>
-                Election definition is loaded.
+                Election Definition is loaded.
               </Text>{' '}
               <Button danger small onPress={unconfigure}>
                 Unconfigure Machine
               </Button>
             </p>
           ) : isFetchingElection ? (
-            <p>Loading Election Definition from Admin Card…</p>
+            <p>Loading Election Definition from Election Manager card…</p>
```

Comment:
For greater global consistency, started lower casing "card" even after capitalized user roles